examples/ConfigurationStoreBackup/host.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "2.0",
+    "logging": {
```

Comment:
I think we can remove the whole "logging" section. We don't need AppInsights.

Reply:
done